src/collections/_documentation/platforms/javascript/electron.md

```diff
@@ -5,6 +5,46 @@ sidebar_order: 2000
 
 `@sentry/electron` is the official Sentry SDK for Electron applications. It can capture JavaScript exceptions in the main process and renderers, as well as collect native crash reports (Minidumps).
 
+## Using it together with Webpack
```

Comment:
Do we want to put this that high?

Asking because this is the first thing people see after setup, not sure if it is the best UX.

Reply:
This is still the style of docs where the entry point is this page
https://docs.sentry.io/error-reporting/quickstart/?platform=electron

and this only special cases for the SDK.
Since Webpack is often used I think it fair / doesn't matter that much.

Reply:
IMO this should be below `## Configuring the Client` as it expects the user to have already installed `@sentry/electron`.